dts/bindings/serial/realtek,rts5912-uart.yaml

```diff
@@ -22,3 +22,6 @@ properties:
 
   pinctrl-names:
     required: true
+
+  gpios:
```

Comment:
This should be `receive-gpios` or `rx-gpios`.  The property also needs a description field specifying how this property is used.